qless/__init__.py

```diff
@@ -129,6 +132,8 @@ def __init__(self, url='redis://localhost:6379', hostname=None, **kwargs):
         import socket
         # This is our unique idenitifier as a worker
         self.worker_name = hostname or socket.gethostname()
+        if PY3:
+            kwargs['decode_responses'] = True
```

Comment:
I wonder which attributes of jobs actually allow binary data.

In the Python bindings, the free-form `data` attribute is always json-encoded, and therefore is always a valid utf-8 string.

The rest of the string attributes might contain non-utf-8 data (jid, worker, klass, ...), but I believe it's safe to assume that they shouldn't.